classes/order/OrderState.php

```diff
@@ -162,4 +162,21 @@ public function isRemovable()
     {
         return !($this->unremovable);
     }
+
+    /**
+     * @param string $name
+     * @param int $idLang
+     * @param int|null $idOrderState
+     *
+     * @return int
+     */
+    public static function countByLocalizedName(string $name, int $idLang, ?int $idOrderState): int
+    {
+        return (int) DB::getInstance(_PS_USE_SQL_SLAVE_)->getValue(
+            'SELECT COUNT(*) AS count FROM ' . _DB_PREFIX_ . 'order_state_lang' .
+            ' WHERE id_lang = ' . $idLang .
+            ' AND name = "' . $name . '"' .
```

Comment:
```suggestion
            ' AND name = \'' . pSQL($name) . '\'' .
```
I think we need to escape this string 🤔  ?

Reply:
Good idea